# Generated by Django 2.1.15 on 2021-01-21 19:38

from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('faults', '0002_auto_20210119_1438'),
    ]

    operations = [
        migrations.AlterModelOptions(
            name='fault',
            options={'ordering': ('-occurred',), 'permissions': (('can_review', 'Can review faults'),)},
        ),
        migrations.AlterField(
            model_name='fault',
            name='occurred',
            field=models.DateTimeField(db_index=True, default=django.utils.timezone.now, help_text='When did this fault occur. Format DD MMM YYYY hh:mm (hh:mm is 24h time e.g. 31 May 2012 14:30)', verbose_name='Date & Time fault occurred'),
        ),
        migrations.AlterField(
            model_name='faulttype',
            name='description',
            field=models.TextField(blank=True, help_text='Enter a description for this fault type', verbose_name='description'),
        ),
        migrations.AlterField(
            model_name='faulttype',
            name='slug',
            field=models.SlugField(editable=False, help_text='Unique URL friendly identifier made of lowercase characters, dashes, and underscores.', max_length=255, unique=True),
        ),
    ]
